<ui version="4.0" >
 <class>PatientDialog</class>
 <widget class="QDialog" name="PatientDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Patient Information</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>15</y>
     <width>776</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="2" >
     <widget class="QLabel" name="textLabel1" >
      <property name="text" >
       <string>&amp;Last Name</string>
      </property>
      <property name="buddy" >
       <cstring>lastNameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="5" >
     <widget class="QLineEdit" name="lastNameLineEdit" />
    </item>
    <item row="0" column="7" colspan="2" >
     <widget class="QLabel" name="textLabel2" >
      <property name="text" >
       <string>&amp;First Name</string>
      </property>
      <property name="buddy" >
       <cstring>firstNameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="9" colspan="3" >
     <widget class="QLineEdit" name="firstNameLineEdit" />
    </item>
    <item row="0" column="12" colspan="4" >
     <widget class="QLabel" name="textLabel3" >
      <property name="text" >
       <string>&amp;Middle Name</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignVCenter</set>
      </property>
      <property name="buddy" >
       <cstring>middleNameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="16" colspan="3" >
     <widget class="QLineEdit" name="middleNameLineEdit" />
    </item>
    <item row="1" column="0" colspan="3" >
     <widget class="QLabel" name="textLabel2_2" >
      <property name="text" >
       <string>M&amp;RN</string>
      </property>
      <property name="buddy" >
       <cstring>mrnLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="5" >
     <widget class="QLineEdit" name="mrnLineEdit" />
    </item>
    <item row="1" column="9" >
     <widget class="QLabel" name="textLabel6" >
      <property name="text" >
       <string>&amp;Height</string>
      </property>
      <property name="buddy" >
       <cstring>heightInLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="10" colspan="4" >
     <widget class="QLineEdit" name="heightInLineEdit" />
    </item>
    <item row="1" column="14" >
     <widget class="QLabel" name="textLabel1_3" >
      <property name="text" >
       <string>in</string>
      </property>
     </widget>
    </item>
    <item row="1" column="15" colspan="4" >
     <widget class="QLineEdit" name="heightCmLineEdit" />
    </item>
    <item row="1" column="19" >
     <widget class="QLabel" name="textLabel3_2" >
      <property name="text" >
       <string>cm</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3" >
     <widget class="QLabel" name="textLabel1_5" >
      <property name="text" >
       <string>Study &amp;Date</string>
      </property>
      <property name="buddy" >
       <cstring>studyDateTimeEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="3" >
     <widget class="QDateTimeEdit" name="studyDateTimeEdit" />
    </item>
    <item row="2" column="9" >
     <widget class="QLabel" name="textLabel11" >
      <property name="text" >
       <string>&amp;Weight</string>
      </property>
      <property name="buddy" >
       <cstring>weightLbsLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="10" colspan="4" >
     <widget class="QLineEdit" name="weightLbsLineEdit" />
    </item>
    <item row="2" column="14" >
     <widget class="QLabel" name="textLabel2_3" >
      <property name="text" >
       <string>lbs</string>
      </property>
     </widget>
    </item>
    <item row="2" column="15" colspan="4" >
     <widget class="QLineEdit" name="weightKgLineEdit" />
    </item>
    <item row="2" column="19" >
     <widget class="QLabel" name="textLabel4_2" >
      <property name="text" >
       <string>kg</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="textLabel2_4" >
      <property name="text" >
       <string>Study &amp;#</string>
      </property>
      <property name="buddy" >
       <cstring>studyNumberLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="6" >
     <widget class="QLineEdit" name="studyNumberLineEdit" />
    </item>
    <item row="3" column="8" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="9" >
     <widget class="QLabel" name="textLabel12" >
      <property name="text" >
       <string>&amp;BSA</string>
      </property>
      <property name="buddy" >
       <cstring>bsaLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="10" colspan="4" >
     <widget class="QLineEdit" name="bsaLineEdit" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="17" >
     <widget class="QLabel" name="textLabel5_2" >
      <property name="text" >
       <string>m&lt;SUP>2&lt;/SUP></string>
      </property>
     </widget>
    </item>
    <item row="3" column="18" colspan="2" >
     <widget class="QCheckBox" name="manualEditBsaCheckBox" >
      <property name="text" >
       <string>Manual &amp;Edit</string>
      </property>
      <property name="shortcut" >
       <string>Alt+E</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="5" >
     <widget class="QLabel" name="textLabel4" >
      <property name="text" >
       <string>Date of Birth</string>
      </property>
      <property name="buddy" >
       <cstring>dobDateEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="6" colspan="3" >
     <widget class="QDateEdit" name="dobDateEdit" />
    </item>
    <item row="4" column="9" colspan="2" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="11" colspan="5" >
     <widget class="QLabel" name="textLabel1_4" >
      <property name="text" >
       <string>&amp;Gender</string>
      </property>
      <property name="buddy" >
       <cstring>sexComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="16" colspan="4" >
     <widget class="QComboBox" name="sexComboBox" >
      <item>
       <property name="text" >
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Female</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0" colspan="4" >
     <widget class="QLabel" name="textLabel5" >
      <property name="text" >
       <string>&amp;Heart</string>
      </property>
      <property name="buddy" >
       <cstring>heartComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="3" >
     <widget class="QComboBox" name="heartComboBox" >
      <item>
       <property name="text" >
        <string>default</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>new heart</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="7" colspan="4" >
     <widget class="QLabel" name="textLabel13" >
      <property name="text" >
       <string>&amp;Vagal Tone</string>
      </property>
      <property name="buddy" >
       <cstring>vagalToneSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="11" colspan="2" >
     <widget class="QSpinBox" name="vagalToneSpinBox" >
      <property name="minimum" >
       <number>1</number>
      </property>
      <property name="maximum" >
       <number>100</number>
      </property>
      <property name="value" >
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="5" column="13" colspan="4" >
     <widget class="QLabel" name="textLabel14" >
      <property name="text" >
       <string>S&amp;ympathetic Tone</string>
      </property>
      <property name="buddy" >
       <cstring>sympatheticToneSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="17" colspan="3" >
     <widget class="QSpinBox" name="sympatheticToneSpinBox" >
      <property name="minimum" >
       <number>1</number>
      </property>
      <property name="maximum" >
       <number>100</number>
      </property>
      <property name="value" >
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="5" >
     <widget class="QLabel" name="textLabel1_2" >
      <property name="text" >
       <string>E&amp;jection Fraction</string>
      </property>
      <property name="buddy" >
       <cstring>efSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="5" colspan="3" >
     <widget class="QSpinBox" name="efSpinBox" >
      <property name="minimum" >
       <number>1</number>
      </property>
      <property name="maximum" >
       <number>100</number>
      </property>
      <property name="value" >
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="6" column="8" >
     <widget class="QLabel" name="textLabel3_3" >
      <property name="text" >
       <string>(%)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="9" colspan="10" >
     <widget class="QCheckBox" name="ischemiaCheckBox" >
      <property name="text" >
       <string>&amp;Ischemia</string>
      </property>
      <property name="shortcut" >
       <string>Alt+I</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="20" >
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOk" >
        <property name="text" >
         <string>&amp;Save</string>
        </property>
        <property name="shortcut" >
         <string>Alt+S</string>
        </property>
        <property name="autoDefault" >
         <bool>true</bool>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel" >
        <property name="text" >
         <string>&amp;Cancel</string>
        </property>
        <property name="shortcut" >
         <string/>
        </property>
        <property name="autoDefault" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>lastNameLineEdit</tabstop>
  <tabstop>firstNameLineEdit</tabstop>
  <tabstop>middleNameLineEdit</tabstop>
  <tabstop>mrnLineEdit</tabstop>
  <tabstop>heightInLineEdit</tabstop>
  <tabstop>heightCmLineEdit</tabstop>
  <tabstop>studyDateTimeEdit</tabstop>
  <tabstop>weightLbsLineEdit</tabstop>
  <tabstop>weightKgLineEdit</tabstop>
  <tabstop>studyNumberLineEdit</tabstop>
  <tabstop>bsaLineEdit</tabstop>
  <tabstop>manualEditBsaCheckBox</tabstop>
  <tabstop>dobDateEdit</tabstop>
  <tabstop>sexComboBox</tabstop>
  <tabstop>heartComboBox</tabstop>
  <tabstop>vagalToneSpinBox</tabstop>
  <tabstop>sympatheticToneSpinBox</tabstop>
  <tabstop>efSpinBox</tabstop>
  <tabstop>ischemiaCheckBox</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>PatientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>661</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel" >
     <x>153</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>PatientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>764</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel" >
     <x>480</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
